apps/journeys-admin-e2e/src/pages/card-level-actions.ts

```
@@ -1497,9 +1489,33 @@ export class CardLevelActionPage {
+    ]
+
+    let found = false
+    for (const selector of chevronDownSelector) {
+      try {
+        await expect(this.page.locator(selector)).toBeVisible({ timeout: 3000 })
+        await this.page.locator(selector).click()
+        found = true
+        break
+      } catch (error) {
+        // Try next selector
+        continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)